edx_proctoring/views.py

```diff
@@ -422,12 +422,18 @@ def put(self, request, attempt_id):
                     exam_id=attempt['proctored_exam']['id'],
                     user_id=request.user.id
                 )
-            elif action == 'submit':
+            elif action == 'timed_submit' or action == 'client_close':
                 exam_attempt_id = update_attempt_status(
                     attempt['proctored_exam']['id'],
```

Comment:
There used to be one `submit` which I now have split into `timed_submit` and `proctored_submit` depending on whether it's a timed exam or a proctored exam, because the transitions after a timed exam behave slightly differently than the transitions after a proctored exam now.